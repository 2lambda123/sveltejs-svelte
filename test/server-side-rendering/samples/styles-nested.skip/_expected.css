div.svelte-awisl7{color:red}
div.svelte-1us38kz{color:green}