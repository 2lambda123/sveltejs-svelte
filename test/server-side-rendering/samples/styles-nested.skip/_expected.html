<div class="svelte-awisl7">red</div>
<div class="svelte-1us38kz">green: foo</div>



<div class="svelte-1us38kz">green: bar</div>


